### PR TITLE
feat(transaction): parse all fungible asset transfer function variants

### LINKED ISSUE #1157: [Feature Request] parse transfer_fungible_assets
# 🚀 Feature Request Description

For 1-to-1 transfers, typically the sender and receiver are parsed out

However, for the following transaction, the receiver is not parsed out

https://explorer.aptoslabs.com/txn/2946262614?network=mainnet

## Expected Behavior

Receiver is parsed out

### PR DESCRIPTION
**Pull request by gregnazario**
### Description

Addresses the feature request to parse `transfer_fungible_assets` and related fungible asset transfer functions so the receiver is displayed and the transfer action is shown in the transaction overview.

**Changes:**

1. **Receiver parsing (`getTransactionCounterparty`)** — Added support for all FA transfer function variants:
   - `0x1::fungible_asset::transfer` — store-to-store transfer; receiver resolved by looking up the destination store's `ObjectCore` owner from the transaction's write-set changes
   - `0x1::dispatchable_fungible_asset::transfer` — same as above (dispatch variant)
   - `0x1::aptos_account::batch_transfer_fungible_assets` — single-recipient batch shows the receiver; multi-recipient batches fall through to smart contract display

2. **Fungible asset transfer action parsing** — Added a new `FungibleAssetTransfer` action type to the `TransactionActionsRow` component. Since FA transfer functions don't emit transfer events, we extract sender, receiver, asset metadata, and amount directly from the entry function payload arguments. Handles:
   - `0x1::aptos_account::transfer_fungible_assets`
   - `0x1::primary_fungible_store::transfer`
   - `0x1::fungible_asset::transfer` (resolves store owner + metadata from changes)
   - `0x1::dispatchable_fungible_asset::transfer` (same)
   - `0x1::aptos_account::batch_transfer_fungible_assets` (generates one action per recipient)

3. **Comprehensive unit tests** — 20 test cases for `getTransactionCounterparty` covering all transfer types, store-to-store transfers, batch transfers, multisig payloads, and edge cases.

**Verified with transaction [2946262614](https://explorer.aptoslabs.com/txn/2946262614?network=mainnet):**
- Receiver at `arguments[1]` = `0xb5fcaf97...` is correctly parsed
- Asset metadata at `arguments[0]` = `{inner: "0x357b0b74..."}` is correctly extracted
- Amount `15021000000` is displayed with proper decimal formatting

### Related Links

- Closes #1157
- Related (closed, not merged): #1170, #1184

### Checklist

- [x] `pnpm fmt` passes
- [x] `pnpm lint` passes (TypeScript + Biome)
- [x] `pnpm test --run` passes (126 tests, 7 files)

<div><a href="https://cursor.com/agents/bc-5aaa3210-b060-4987-9498-30471f2ed6be"><picture><source media="(prefers-color-scheme: dark)" srcset="https://cursor.com/assets/images/open-in-web-dark.png"><source media="(prefers-color-scheme: light)" srcset="https://cursor.com/assets/images/open-in-web-light.png">Open in Web</picture></a>&nbsp;<a href="https://cursor.com/background-agent?bcId=bc-5aaa3210-b060-4987-9498-30471f2ed6be"><picture><source media="(prefers-color-scheme: dark)" srcset="https://cursor.com/assets/images/open-in-cursor-dark.png"><source media="(prefers-color-scheme: light)" srcset="https://cursor.com/assets/images/open-in-cursor-light.png">Open in Cursor</picture></a>&nbsp;</div>

### CODE REVIEW

File: app/pages/Transaction/Tabs/UserTransactionOverviewTab.tsx [hunk 1]
```diff
@@ -443,6 +452,84 @@ const parsers = [
...
+  "0x1::aptos_account::transfer_fungible_assets": {
+    metadataIndex: 0,
+    recipientIndex: 1,
+    amountIndex: 2,
+  },
+  "0x1::primary_fungible_store::transfer": {
+    metadataIndex: 0,
+    recipientIndex: 1,
+    amountIndex: 2,
+  },
```
Comment (gregnazario) [inline app/pages/Transaction/Tabs/UserTransactionOverviewTab.tsx]: What about `0x1::fungible_asset::transfer`?  Or `0x1::dispatchable_fungible_asset::transfer`?  Or `0x1::aptos_account::batch_transfer_fungible_assets`
